plopComponents/application/prompts.js

```diff
@@ -1,111 +1,81 @@
+          value: "searchSummary"
+        },
+        {
+          name: "Question and Answer | Expects the user to ask a question and provides them a concise answer.",
+          value: "questionAndAnswer"
+        }
       ]
     });
 
-    const isDemoUi = appTypeAns.appType === "preset";
-
-    const presetAppDirNameAns = await inquirer.prompt({
-      when: () => isDemoUi,
+    const dataSourceAns = await inquirer.prompt({
       type: "list",
-      name: "presetAppDirName",
-      message: "Choose a pre-built sample UI.",
+      name: "dataSource",
+      message:
+        "Want to connect to your own data or use our sample data? Sample data consists of pages scraped from docs.vectara.com.",
       choices: [
+        { name: "Use my own data", value: "customData" },
         {
-          name: "Vectara Docs - Answer questions about Vectara's platform documentation",
-          value: "vectara-docs"
-        },
-        {
-          name: "Vectara.com Q&A - Answer questions about Vectara's company website",
-          value: "vectara-website"
-        },
-        {
-          name: "Ask Feynman - Answer questions about Richard Feynman's lectures",
-          value: "ask-feynman"
+          name: "Use the Vectara Docs sample data",
+          value: "sampleData"
         }
       ]
     });
 
-    const customAppDirNameAns = await inquirer.prompt({
-      when: () => !isDemoUi,
-      type: "input",
-      name: "customAppDirName",
-      message: "What directory name would you like to use?"
-    });
+    const isCustomData = dataSourceAns.dataSource === "customData";
 
     const appNameAns = await inquirer.prompt({
-      when: () => !isDemoUi,
+      when: () => isCustomData,
       type: "input",
       name: "appName",
-      message: "What would you like to name your application?"
+      message: "What do you want to name your application?"
     });
 
     const customerIdAns = await inquirer.prompt({
-      when: () => !isDemoUi,
+      when: () => isCustomData,
       type: "input",
       name: "customerId",
       message: "What's your Vectara Customer ID?"
     });
 
     const corpusIdAns = await inquirer.prompt({
-      when: () => !isDemoUi,
+      when: () => isCustomData,
       type: "input",
       name: "corpusId",
-      message: "What Vectara Corpus ID is associated with your data?"
+      message: "What's the Corpus ID of the corpus that contains your data?"
     });
 
     const apiKeyAns = await inquirer.prompt({
-      when: () => !isDemoUi,
+      when: () => isCustomData,
       type: "input",
       name: "apiKey",
-      message: "What is your Vectara QueryService API Key (This can be safely shared)?"
+      message: "What's your QueryService API Key? This must have access to the corpus."
     });
 
     const questions = [];
     const haveQuestionsAns = await inquirer.prompt({
-      when: () => !isDemoUi,
+      when: () => isCustomData,
```

Comment:
To address your question re: sample questions, I think what you did here was great. If they're using sample data, we can ask them for custom questions until they're done.

Just clarifying: If they choose to use the sample data set (vectara docs), we should default them to the docs sample questions and note the Feynman ones, right? Just got a little confused since your description mentioned just using the Feynman ones by default, but those wouldn't really apply to data in the docs corpus.

Reply:
Oh you're right, I think at one point I was using the Feynman data as the single sample, but then I switched over to docs at some point.